### PR TITLE
Use `printf` instead of `echo -n` for better portability

### PR DESCRIPTION
**Pull request by devnsec-com**
Related issue: #1236

### CODE REVIEW

File: bootstrap/venv.sh [hunk 1]
```diff
@@ -25,9 +25,9 @@ pip install -U letsencrypt letsencrypt-apache # letsencrypt-nginx
 echo
 echo "Congratulations, Let's Encrypt has been successfully installed/updated!"
 echo
-echo -n "Your prompt should now be prepended with ($VENV_NAME). Next "
-echo -n "time, if the prompt is different, 'source' this script again "
-echo -n "before running 'letsencrypt'."
+printf "%s" "Your prompt should now be prepended with ($VENV_NAME). Next "
```
Comment (kuba) [inline bootstrap/venv.sh]: why do we need "%s" in this case but not others?

**Comment by kuba** [comment]
It seems like `printf` drop quotes: https://travis-ci.org/letsencrypt/letsencrypt/jobs/88451445#L2880

File: bootstrap/venv.sh (hunk 1 above)
Comment (devnsec-com) [inline bootstrap/venv.sh]: Becaus the $VENV_NAME could contains symbols which will cause issues, but others are plain texts.

**Comment by devnsec-com** [comment]
> It seems like printf drop quotes: https://travis-ci.org/letsencrypt/letsencrypt/jobs/88451445#L2880

I'm looking into this issue.

**Comment by devnsec-com** [comment]
Seems like I'm quoting a quoted string which lead to drop quotes. I'm going to push a fixed version soon.

**Comment by devnsec-com** [comment]
@kuba OK, that problem has solved.

**Comment by kuba** [comment]
LGTM

**Pull request merged**